hashes/extended_tests/schemars/README.md

```diff
@@ -0,0 +1,10 @@
+# Test crate for the schemars feature
+
+Run as usual with `cargo test`.
+
+## Minimum Supported Rust Version (MSRV)
+
+To run the tests with the MSRV you will need to pin some dependencies:
+
+- `cargo update -p serde --precise 1.0.156`
+- `cargo update -p syn --precise 1.0.107`
```

Comment:
I think this isn't needed anymore?

Reply:
I'm unsure. The `serde` line definitely is, but `syn` you may be right that we can drop. In any case, can wait for a followup PR.

Reply:
Verified, we no longer need the `syn` pin. Will do as follow up because this PR is blocking other work. Thanks

Reply:
Yep, that's what I meant.